### PR TITLE
Rename POR options switches, purge mixed option

### PR DESCRIPTION
**Pull request by njulrik**
Clean-up of naming around POR methods. Our method is now named `automaton` and is set as default, whereas the one based on Liebke's theory is named `liebke`. The help message reflects this. The mixed option which would apply the classic method in non-reachability state has been purged as the notion of 'reachability state' is obsolete.

### CODE REVIEW

File: include/PetriEngine/options.h [hunk 1]
```diff
@@ -43,9 +43,9 @@ enum class APCompression {
 enum class LTLPartialOrder {
     None,
     Visible,
-    AutomatonReach,
-    VisibleReach,
-    FullAutomaton
+    Automaton,
+    //VisibleAutomaton,
```
Comment (petergjoel) [inline include/PetriEngine/options.h:47]: If it is commented out, remove it.

File: src/PetriEngine/options.cpp [hunk 2]
```diff
@@ -138,16 +138,13 @@ void printHelp() {
         "  --partition-timeout <timeout>        Timeout for color partitioning in seconds (default 5)\n"
         "  -l, --lpsolve-timeout <timeout>      LPSolve timeout in seconds, default 10\n"
         "  -p, --disable-partial-order          Disable partial order reduction (stubborn sets)\n"
-        "  --ltl-por <type>                     Select partial order method to use with LTL engine (default reach).\n"
-        "                                       - reach      apply reachability stubborn sets in Büchi states\n"
-        "                                                    that represent reachability subproblems,\n"
-        "                                       - classic    classic stubborn set method.\n"
+        "  --ltl-por <type>                     Select partial order method to use with LTL engine (default automaton).\n"
+        "                                       - automaton  apply Büchi-guided stubborn set method.\n"
```
Comment (petergjoel) [inline src/PetriEngine/options.cpp:142]: Add paper ref like the others.

**Review by petergjoel — CHANGES_REQUESTED** [review]
Remember to update MCC competition script also (if needed).

**Review by srba — CHANGES_REQUESTED** [review]
also the tapaal.sh competition script in the MCC22 directory should be updated with the new switches

**Comment by njulrik** [comment]
I updated the flag on #24, this should be good to go now.

File: src/PetriEngine/options.cpp [hunk 3]
```diff
@@ -462,12 +459,12 @@ bool options_t::parse(int argc, const char** argv) {
...
             } else if (std::strcmp(argv[i + 1], "classic") == 0) {
                 ltl_por = LTLPartialOrder::Visible;
-            } else if (std::strcmp(argv[i + 1], "reach") == 0) {
-                ltl_por = LTLPartialOrder::AutomatonReach;
-            } else if (std::strcmp(argv[i + 1], "mix") == 0) {
-                ltl_por = LTLPartialOrder::VisibleReach;
             } else if (std::strcmp(argv[i + 1], "automaton") == 0) {
-                ltl_por = LTLPartialOrder::FullAutomaton;
+                ltl_por = LTLPartialOrder::Automaton;
+            } else if (std::strcmp(argv[i + 1], "mix") == 0) {
```
Comment (srba) [inline src/PetriEngine/options.cpp:464]: @njulrik, why do you compare here against "mix", it is not mentioned in the options anymore?

**Comment by petergjoel** [comment]
Abandoned, changes merged on top of #24

**Pull request closed without merging**